exchanges/stream/buffer/buffer.go

```diff
@@ -204,7 +205,12 @@ func (w *Orderbook) Update(u *orderbook.Update) error {
 	// A nil ticker means that a zero publish period has been set and the entire
 	// websocket updates will be sent to the engine websocket manager for
 	// display purposes. Same as being verbose.
-	w.dataHandler <- book.ob
+	select {
+	// Needs to be in select case as this can cause a knock on affect with
+	// websocket update and processing times.
+	case w.dataHandler <- book.ob:
+	default:
```

Comment:
[You have changed things here](https://github.com/thrasher-corp/gocryptotrader/pull/1316/files#diff-e1c6631789757e78b4b9024520a74b7b1b6675b49f4e454f0cc147c220ee1a86R216). Is it a bug to merge this orderbook dropping behaviour since its not relevant for this PR?